Libs/Widgets/Testing/Cpp/ctkPathLineEditTest1.cpp

```diff
@@ -42,7 +42,9 @@ int ctkPathLineEditTest1(int argc, char * argv [] )
   ctkPathLineEdit button2("Files",
                           QStringList() << "Images (*.png *.jpg)" << "Text (*.txt)",
                           ctkPathLineEdit::Files);
-  ctkPathLineEdit button3("Dirs", QStringList("CTK*"));
+  ctkPathLineEdit button3("Dirs", QStringList(), ctkPathLineEdit::Dirs);
+  button3.setShowHistoryButton(false);
+  button3.setCurrentPath("c:/tmp");
```

Comment:
What about using:
```
QString QDir::tempPath()
```

Reply:
We use tempPath in one of the path selectors. I'll use `QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation)`.